resolwe/flow/static/flow/processorSchema.json

```diff
@@ -39,6 +39,11 @@
             "type": "string",
             "pattern": "^RAW|CACHED|TEMP$"
         },
+        "scheduling_class": {
+            "description": "Scheduling class",
+            "type": "string",
+            "enum": ["interactive", "batch"]
```

Comment:
Options in `persistence` are enforced with `"pattern": "^RAW|CACHED|TEMP$"`. Make this consistent.